kubernetes-maven-plugin/plugin/src/main/java/org/eclipse/jkube/maven/plugin/mojo/build/ApplyMojo.java

```diff
@@ -140,6 +135,12 @@
     @Parameter(property = "jkube.environment")
     private String environment;
 
+    /**
+     * Namespace to use when accessing Kubernetes or OpenShift
+     */
+    @Parameter(property = "jkube.namespace")
+    protected String namespace;
```

Comment:
Shall we move this to AbstractJKubeMojo?

Reply:
I'm not sure that this applies to **all** mojos (e.g. build).
Currently, each Mojo has its own way of handling namespace.
So, I agree we should consider this, but not in the scope of this PR.

Reply:
ohk